packages/invoices/src/OutboundShipment/DetailView/api.ts

```diff
@@ -1,258 +0,0 @@
-import {
```

Comment:
🤔  moved elsewhere?

Reply:
Ah yes this was moved previously and forgot to remove this